design docs/durandal play by play.txt

```diff
@@ -0,0 +1,55 @@
+Gameplay Example
+
+Player1 joins the server and enters the customization screen.
+
+They choose a government entity with specializations:
+
+- Large Space Agency: Large budget, extensive existing responsibilities.
+- Old Space Agency: Experienced crews and engineers; slower adoption of new technologies and penalties when restructuring or significantly changing processes.
+- Crewed Missions Specialization: Easier and safer crewed missions but significantly increased difficulty and cost for autonomous missions.
+
+Entering the game, Player1 reviews their agency's current obligations: a schedule of commercial satellite launches and ISS-2 service missions. The budget overview indicates a surplus sufficient to consider new projects.
+
+Player1 browses available contractors and finds a rocket capable of carrying two of their regular payloads simultaneously. The contract offers cost efficiency but significantly reduces customization and oversight of safety standards, risking potential mission failures and lower reputation gains if accidents occur.
+
+After careful consideration, Player1 signs the contract. They experience reduced production overhead but retain significant overhead costs due to their intertwined internal structures.
+
+With commercial satellites bundled, launches are freed for a new project. Though a moon mining project is desirable, the budget remains insufficient. Player1 opts for an intermediate project: constructing additional modules for the ISS-2 to rent to corporations for increased revenue.
+
+Player1 instructs available engineers to design modules optimized for tourism. Leveraging their specialization, the design is efficiently completed. Player1 accumulates some debt to launch these modules. Corporations submit bids with various requirements, influencing long-term relationships and reputation. Player1 accepts a bid offering substantial payment and operational integration with Player1's existing infrastructure, effectively creating a profitable space hotel.
+
+Due to high visibility and successful operation management, Player1's popularity significantly rises, directly increasing government funding. This enhanced reputation also attracts better future partnership bids and improved loan terms.
+
+With improved funding, the moon mining operation becomes feasible. However, significant initial funding gaps and no available launches require extensive restructuring. Due to the agency's specialization as an Old Space Agency, the restructuring process incurs additional costs and complexity because of internal resistance and inflexibility. Player1 privatizes satellite launch operations, clearly losing substantial experience in rocketry, requiring considerable time, investment, and effort to recover lost capabilities and integrate new operational frameworks.
```

Comment:
Funding (to me at least) would typically mean external funding as opposed to having the resources available for the operation. Is this intentional?